dbms/src/Operators/OperatorPipeline.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
As this is a runtime info for a specific `Pipeline`, why not call it `PipelineExec`?

The same for `OperatorPipelineBuilder`.

Reply:
ok, updated.